raiden/tests/utils/geth.py

```diff
@@ -90,7 +101,7 @@ def geth_to_cmd(node, datadir, verbosity):
         '--rpcapi', 'eth,net,web3',
         '--rpcaddr', '0.0.0.0',
         '--networkid', '627',
-        '--verbosity', str(verbosity),
+        '--verbosity', '3',
```

Comment:
Why this change? If it's not needed pls remove the `verbosity`  parameter from the function